src/main/asciidoc/specifications.adoc

```diff
@@ -12,10 +12,10 @@
 The following specifications for JDO are available
 
 * The
-http://svn.apache.org/viewvc/db/jdo/trunk/specification/OOO/JDO-3.1.pdf?view=co[JDO
+https://github.com/apache/db-jdo/tree/master/specification/OOO/JDO-3.1.pdf?view=co[JDO
```

Comment:
As a minor note in case we keep the current links: I don't think the ending `?view=co` does anything for PDFs in the GitHub view. It is automatically dropped for me when following the link.